stan/math/prim/fun/gaus_interp.hpp

```diff
@@ -0,0 +1,167 @@
+#ifndef STAN_MATH_PRIM_FUN_GAUS_INTERP
+#define STAN_MATH_PRIM_FUN_GAUS_INTERP
+
+#include <stan/math/prim/fun/conv_gaus_line.hpp>
+#include <cmath>
+#include <vector>
+#include <algorithm>
+
+using namespace std;
+
+namespace stan {
+namespace math {
+
+// set tolerance for how close interpolated values need to be to
+// the specified function values
+const double INTERP_TOL = 1e-8;
+const double SIG2_SCALE = 0.1;
```

Comment:
If this is specific to a function they should be in the function instead of floating in the stan math namespace